src/style-sheets/generic/_button.scss

```diff
@@ -1,32 +1,42 @@
 .button {
-    width: 189px;
-    height: 49px;
-    border-radius: 4px;
-    border-width: 0;
+  width: 189px;
+  height: 49px;
+  border-radius: 4px;
+  border-width: 0;
 
-    font-size: 1.25rem;
-    cursor: pointer;
+  font-size: 1.25rem;
+  cursor: pointer;
 }
 
 .primary {
-    background-color: $olLTAccent;
-    color: $white;
-    font-family: $olBaseFont;
+  background-color: $olLTAccent;
+  color: $white;
+  font-family: $olBaseFont;
+}
+
+.primary:hover {
+  background-color: #259333;
```

Comment:
with colors we will want to use variables. Could you create variables for the hover background color for primary, and the click colors. Something along the lines of olLTPrimaryBtnHover, olLTPrimaryBtnClicked, olLTPrimaryBtnTxtClicked